### ISSUE TITLE
Hindi Translation Progress

### ISSUE THREAD

**Issue by tesseralis** [reactjs/hi.react.dev#1]
Translation details of beta site is available here https://github.com/reactjs/hi.reactjs.org/issues/168
We are not translating these pages anymore.

## Maintainer List

* @saranshkataria
* @arshadkazmi42
## For New Translators

To translate a page:

1. Check that no one else has claimed your page in the checklist and comments below.
2. Comment below with the name of the page you would like to translate. **Please take only one page at a time**.
3. Clone this repo, translate your page, and submit a pull request!

Before contributing, read the glossary and style guide (once they exist) to understand how to translate various technical and React-specific terms.

Please be prompt with your translations! If you find find that you can't commit any more, let the maintainers know so they can assign the page to someone else.

## For Maintainers

When someone volunteers, edit this issue with the username of the volunteer, and with the PR. Ex:

* [X] Home Page (@saranshkataria) #3 #6

When PRs are merged, make sure to mark that page as completed!

## Core Pages

To do before releasing as an "official" translation. **Please translate these pages first.**

* [X] Home Page (@saranshkataria ) #3 #6 
* [x] Tutorial (@arshadkazmi42 ) #8 

### Basics

* [x] Hello World (@gurjotsingh4398 ) #13 
* [ ] Introducing JSX (@arshadkazmi42 ) #12 
* [x] Rendering Elements (@AbhishekCode && @saranshkataria ) #21 #41 
* [x] Components and Props (@helios1101 ) #24 
* [ ] State and Lifecycle #26 
* [ ] Handling Events (@yashguptaz ) #27 
* [ ] Conditional Rendering (@kambleaa007) #43 
* [x] Lists and Keys (@anchal7299 ) #25 
* [ ] Forms (@aashirwadgupta  ) #22 + (New PR) 
* [ ] Lifting State Up (@SambhavSacheti ) #30 
* [x] Composition vs Inheritance (@apoorvtomar2222) #28 
* [ ] Thinking in React (@mayank-gangwal )

### API Reference

* [ ] React (@Hrshk15 )
* [ ] React.Component #76 
* [ ] ReactDOM (@charBap ) #31 
* [ ] ReactDOMServer (@neeraj-here)
* [ ] DOM Elements (@rao123dk ) 
* [x] SyntheticEvent (@ps0305 ) #66 
* [x] Test Utilities (@paramaggarwal ) #45 
* [x] Shallow Renderer (@vermasachin && @arshadkazmi42  ) #44 
* [ ] Test Renderer (@tanishqchamola )
* [x] JS Environment Requirements (@AlokTakshak) #47
* [ ] Glossary (@AlokTakshak) #51 

### Navigation

These are the navigation links that appears in the sidebar. Possibly wait until
the corresponding sections are translated to do these.

* [ ] tutorial/nav.yml
* [ ] docs/nav.yml

## Next Steps

These are the next most important translations, ordered by importance:

### Installation

* [x] Getting Started (@yashguptaz ) #17 
* [ ] Add React to a Website (@NorinMp143 ) #65 
* [ ] Create a New React App (@vishalsodani ) #52 
* [x] CDN Links (@AninditaBasu ) #53

### Hooks

Everyone is hooked on hooks! People are really excited for these and it would be great for them to be translated.

  * [ ] Introducing Hooks (@shob0 )
  * [x] Hooks at a Glance (@gautam-pahuja ) #58 
  * [ ] Using the State Hook (@AlokTakshak )
  * [ ] Using the Effect Hook (@anoopsingh1996 )
  * [ ] Rules of Hooks (@chandrakantnaik789 )
  * [ ] Building Your Own Hooks (@visshaljagtap ) #68 
  * [ ] Hooks API Reference
  * [ ] Hooks FAQ (@ps0305 )

### Warnings

These are the pages that you get when you click the links in the console (e.g. https://reactjs.org/warnings/dont-call-proptypes.html). People tend to search these, so **please don't translate the error message itself**.
 
* [ ] dont-call-proptypes (@ShivamPorwal02 )
* [ ] invalid-aria-prop
* [ ] invalid-hook-call-warning
* [ ] legacy-factories
* [ ] refs-must-have-owner
* [ ] special-props
* [ ] unknown-prop

### Advanced Guides

* [ ] Accessibility
* [ ] Code-Splitting
* [ ] Context
* [ ] Error Boundaries
* [ ] Forwarding Refs
* [ ] Fragments
* [ ] Higher-Order Components
* [ ] Integrating with Other Libraries
* [ ] JSX In Depth (@vishalnagda1 )
* [ ] Optimizing Performance
* [x] Portals (@mayankverr  ) #73 
* [ ] React Without ES6 (@kumari1224 )
* [ ] React Without JSX (@vishalnagda1)
* [ ] Reconciliation
* [ ] Refs and the DOM
* [ ] Render Props
* [ ] Static Type Checking
* [ ] Strict Mode
* [ ] Typechecking With PropTypes
* [ ] Uncontrolled Components
* [ ] Web Components

### FAQ

* [ ] APIs and AJAX (@helios1101 )
* [x] Babel, JSX, and Build Steps (@shobhitsingh29 ) #50 
* [ ] Passing Functions to Components (@nnhhiilliisstt )
* [ ] Component State (@bhatvikrant )
* [ ] Styling and CSS (@PriyankaCodes99)
* [ ] File Structure (@anchal7299 ) #54 
* [ ] Versioning Policy
* [ ] Virtual DOM and Internals (@bhatvikrant)

### Contributing

Priority: Low

* [ ] How to Contribute
* [ ] Codebase Overview
* [ ] Implementation Notes
* [ ] Design Principles

### Components

Components in `src/components` that have some text in them.

* [ ] CodeEditor
* [ ] CodeExample
* [ ] ErrorDecoder
* [ ] LayoutFooter
* [ ] LayoutHeader
* [ ] MarkdownPage

## Additional Translations

These are not the primary translation targets.

* [ ] Blog
* [ ] Community

**Comment by arshadkazmi42** [comment]
@saranshkataria I am starting with Tutrial page

**Comment by apoorvtomar2222** [comment]
@tesseralis  @arshadkazmi42  @saranshkataria  

How can we differentiate that what all page among the list have already been done ?

**Comment by arshadkazmi42** [comment]
@apoorvtomar2222 If any page is completed, it will has a check mark like below
- [x] Done
And if its in progress it will have the username of contributor who is working on it like below
- [] Page (@arshadkazmi42 )

**Comment by arshadkazmi42** [comment]
if you would like to work on any of the page translation you can comment here with the page name and we will assign that page to you.

**Comment by saranshkataria** [comment]
@arshadkazmi42 I am picking up the home page. @apoorvtomar2222 feel free to pick anything else apart from the homepage or tutorial. I think it would be great if we go sequentially though.

**Comment by gurjotsingh4398** [comment]
I would like to work on Hello World page.

**Comment by arshadkazmi42** [comment]
Go ahead

**Comment by arshadkazmi42** [comment]
@saranshkataria Home Page is still not completely translated. I have unmarked it as done.
You can try running it locally. These are the subtitles which are not translated. 
- A Simple Component
- A Stateful Component
- An Application
- A Component Using External Plugins

We will mark it as done, when all these are also translated.

**Comment by saranshkataria** [comment]
@arshadkazmi42, I had mentioned in the PR that it is not complete. I wanted to put something in to kick things off. I know about the remaining sections. I had never marked it as done initially because of that. Working on those sections. Thanks though.

**Comment by arshadkazmi42** [comment]
Ok cool. I thought you missed it. 
I think those are part of Home page right? Or do we have separate entries for those in the list?

**Comment by saranshkataria** [comment]
They are in the home page. Also, should we split these into multiple issues instead of just one?

**Comment by arshadkazmi42** [comment]
I think this is fine. To keep track for all the Pages.

Also, I am trying to get more contributors to help us with the translations

**Comment by saranshkataria** [comment]
@arshadkazmi42 the home page is done now! Waiting for you to complete the tutorial page to have the basics done.

**Comment by arshadkazmi42** [comment]
@saranshkataria Awesome.
I am 70% done with the tutorial page. Will send a PR in a day or two

**Comment by hozefaj** [comment]
I can take a crack at the `Warning` section.

**Comment by arshadkazmi42** [comment]
Cool. go ahead. Which warning page you will be working on? Can you pick one, so I can assign your name next to page name

**Comment by hozefaj** [comment]
I can start with `dont-call-proptypes`. Do you have a timeline by which the pages need to get done?

**Comment by arshadkazmi42** [comment]
Cool. I will add your username to that page.
We don't have any fixed timelines, but if you get stuck at anything you can let us know.

**Comment by hozefaj** [comment]
Just curious if there is any long term automation plan to keep the docs in sync with the English version? Or the updates are expected to happen manually every time?

**Comment by tesseralis** [comment]
@hozefaj a bot will create PRs around once a day when it detects changes to master branch, as so:

https://github.com/reactjs/hi.reactjs.org/pull/11

**Comment by PriyankaCodes99** [comment]
Can I translate the Styling and CSS page?

**Comment by saranshkataria** [comment]
@PriyankaCodes99 go for it! Adding your name next to it

**Comment by PriyankaCodes99** [comment]
> 
> 
> @PriyankaCodes99 go for it! Adding your name next to it

Thanks! Will have it done in a couple of days (max).

**Comment by AbhishekCode** [comment]
Hey, Can I translate Rendering Elements page?

**Comment by arshadkazmi42** [comment]
@AbhishekCode go ahead. I will assign your name next to the page

**Comment by AbhishekCode** [comment]
@arshadkazmi42 Thanks 👍

**Comment by helios1101** [comment]
Can I do the translation for Components and Props page?

**Comment by arshadkazmi42** [comment]
@helios1101 Go ahead. I will assign your name next to the page.

**Comment by SahibArora** [comment]
Hi @arshadkazmi42, I can start with reactDOMServer.

**Comment by arshadkazmi42** [comment]
@SahibArora You can work on it. I will assign your name to the page

**Comment by saranshkataria** [comment]
I think we should go in order of the list so that we complete the core pages first. Just a note for people who take up pages in the future

**Comment by arshadkazmi42** [comment]
@saranshkataria I know core pages are important, but we should not force it on contributors, I think we should let contributors pick the pages as their comfort.

**Comment by helios1101** [comment]
> @helios1101 Go ahead. I will assign your name next to the page.

Sure. The work is under progress at #24

**Comment by anchal7299** [comment]
Can i translate Lists and Keys page?

**Comment by arshadkazmi42** [comment]
@anchal7299 sure. Let me assign your name next to the page

**Comment by charBap** [comment]
Can I do State and Lifecycle? and also Handling events?

**Comment by arshadkazmi42** [comment]
@charBap sure. Let me assign your name to the page

**Comment by saranshkataria** [comment]
@charBap @anchal7299 @hozefaj @PriyankaCodes99 any progress?

**Comment by hozefaj** [comment]
Have not been able to make a lot of progress. Its much harder than I thought. Choosing what to translate and keeping context the same is tricky.

**Comment by saranshkataria** [comment]
> Have not been able to make a lot of progress. Its much harder than I thought. Choosing what to translate and keeping context the same is tricky.

Let us know if we can help

**Comment by charBap** [comment]
@saranshkataria I have started working on it. Just figuring out the the words that need to be in English.

**Comment by arshadkazmi42** [comment]
@charBap We have also added a glossary, at #2 
You can check the words we are not translating and we also have some common translations added there

**Comment by PriyankaCodes99** [comment]
> 
> 
> @charBap @anchal7299 @hozefaj @PriyankaCodes99 any progress?

Yes, just need some time to finish it completely. Trying for this weekend.

**Comment by yashguptaz** [comment]
Can you please assign me _Handling Events_
Let's get another page done!

**Comment by arshadkazmi42** [comment]
@yashguptaz Sure. Let me assign your name right away

**Comment by SambhavSacheti** [comment]
Can I please translate Lifting State Up ?

**Comment by saranshkataria** [comment]
@SambhavSacheti sure, putting your name next to it!

**Comment by apoorvtomar2222** [comment]
@saranshkataria @arshadkazmi42 

I will like to work on *Composition vs Inheritance* page. Can you assign this to my name?

**Comment by arshadkazmi42** [comment]
@apoorvtomar2222 Cool. I will assign it right away.

**Comment by realdeepak** [comment]
@saranshkataria @arshadkazmi42 
I would like to translate these pages : these are the part of Installation -

- Add React to a Website
- Create a New React App
- CDN Links

Please assign it on my name.

**Comment by arshadkazmi42** [comment]
@realdeepak We want to proceed page by page. 
Let's start with `Add React to a Website`. I will add your name to this page, once you are done with this, you can ping us here for other pages

**Comment by apoorvtomar2222** [comment]
@arshadkazmi42 @saranshkataria 
Let's mark this as done Composition vs Inheritance?
Can I move forward with "Thinking in React"?

**Comment by arshadkazmi42** [comment]
@apoorvtomar2222 Yes, you can let me assign your name next to `Thinking in React`

**Comment by charBap** [comment]
@arshadkazmi42  Can I take API reference -> ReactDOM ?

**Comment by arshadkazmi42** [comment]
@charBap You have a PR in the review. Let's finish that up and then you can take up the next pages.
As it will be difficult to work on multiple pages at a time since there are a lot of iterations.
If you are still ok working on multiple pages I can assign the page to you, but we will be reviewing the new page once your first PR is reviewed and merged.

**Comment by manavshrivastavagit** [comment]
Hi @arshadkazmi42  Can I translate API Reference -> React?

**Comment by arshadkazmi42** [comment]
@manavshrivastavagit sure. go ahead
i have assigned your name to the page

**Comment by charBap** [comment]
> @charBap You have a PR in the review. Let's finish that up and then you can take up the next pages.
> As it will be difficult to work on multiple pages at a time since there are a lot of iterations.
> If you are still ok working on multiple pages I can assign the page to you, but we will be reviewing the new page once your first PR is reviewed and merged.

@arshadkazmi42  I am ok with working on multiple pages. Since my previous page is in the second stage of review, I would like to start working on this. Thank you.

**Comment by arshadkazmi42** [comment]
@charBap Cool. Let me assign your name to the page

**Comment by kambleaa007** [comment]
Feel free to assign me a page too.
@arshadkazmi42 
Thanks

**Comment by arshadkazmi42** [comment]
@kambleaa007 You can pickup any page which is not assign to anyone.
Once you choose a page I will assign your name next to the page

**Comment by kambleaa007** [comment]
Yeah,
I like choose to do ` Conditional Rendering`
plz assign it to me,
Thanks

**Comment by arshadkazmi42** [comment]
> Yeah,
> I like choose to do ` Conditional Rendering`
> plz assign it to me,
> Thanks

Great. Assigned your name to the page

**Comment by vermasachin** [comment]
@arshadkazmi42 I would like to translate 'Shallow Renderer', seems small enough to begin.

**Comment by arshadkazmi42** [comment]
@vermasachin Cool.
Let me assign your name to that page

**Comment by kambleaa007** [comment]
hello guys, which tools you all are using to translate, i tried with google translator, it seems tough.

**Comment by arshadkazmi42** [comment]
@kambleaa007 We do not just copy paste the translation, we try to do some manual translations.
I personally use Google Translate + [Hindi Typing](http://indiatyping.com/index.php/english-to-hindi-typing)

**Comment by paramaggarwal** [comment]
Please list me for API Reference > Test Utilities. Thanks.

**Comment by arshadkazmi42** [comment]
> Please list me for API Reference > Test Utilities. Thanks.

@paramaggarwal Assigned your name to Test utilities 👍

**Comment by AlokTakshak** [comment]
@arshadkazmi42 i want to start contributing from `JS Environment Requirements` please assign it to me

**Comment by arshadkazmi42** [comment]
@AlokTakshak Assiged your name next to the page.
You can refer #2 during translation.
It has information about the common translations and words which should not be translated

**Comment by AlokTakshak** [comment]
@arshadkazmi42  please assign ` API REFERENCE > Glossary `  to me

**Comment by arshadkazmi42** [comment]
@AlokTakshak done. Assigned the page

**Comment by arshadkazmi42** [comment]
> @arshadkazmi42 can i take on` Babel, JSX, and Build Steps`

Done. Assigned your name to the page

**Comment by helios1101** [comment]
@arshadkazmi42 can I work with APIs and AJAX in FAQ section?

**Comment by shaekhjamal** [comment]
@arshadkazmi42 Can I take `APIs and AJAX` this page?

**Comment by arshadkazmi42** [comment]
@helios1101 Assigned your name next to the page

**Comment by vishalsodani** [comment]
I would like to take on Create a New React App. Is this the page https://hi.reactjs.org/docs/create-a-new-react-app.html? @arshadkazmi42

**Comment by arshadkazmi42** [comment]
> I would like to take on Create a New React App. 

Cool. go ahead. Assigned your name to the page

> Is this the page https://hi.reactjs.org/docs/create-a-new-react-app.html? @arshadkazmi42

Yes. This is the page

**Comment by anchal7299** [comment]
@arshadkazmi42 Can i transalte File Structure page in FAQ section?

**Comment by arshadkazmi42** [comment]
@anchal7299 Sure. Go ahead. Assigned your name next to the page

**Comment by AninditaBasu** [comment]
@arshadkazmi42 
May I do CDN links? https://hi.reactjs.org/docs/cdn-links.html

**Comment by arshadkazmi42** [comment]
> @arshadkazmi42
> May I do CDN links? https://hi.reactjs.org/docs/cdn-links.html

Sure. Go ahead.

**Comment by AlokTakshak** [comment]
@arshadkazmi42  Assign me Hooks at Glance and using State hooks

**Comment by arshadkazmi42** [comment]
> @arshadkazmi42 Assign me Hooks at Glance and using State hooks

@AlokTakshak You already have a PR review in progress. Let's get that closed first then you can take up a new page.

**Comment by rao123dk** [comment]
@arshadkazmi42 I would like to  work on ["DOM Elements" ](https://hi.reactjs.org/docs/dom-elements.html)

**Comment by arshadkazmi42** [comment]
@rao123dk Go ahead. Assigned to name to the page

**Comment by gautam-pahuja** [comment]
@tesseralis  @arshadkazmi42  @saranshkataria 
Hello guys, I would like to work with React hooks topic. I am planning to take 2 topics:
- Hooks at a Glance
- Using the State Hook

Can you guys pin my name on these.

Thanks ❤️

**Comment by arshadkazmi42** [comment]
@gautam-pahuja You can start with one of them, once your first page changes are merged you can start with the second one.
I will assign your name to the first page `Hooks at a Glance`

**Comment by arshadkazmi42** [comment]
@SahibArora removed your name from `Forms` page as your PR #22 is closed due to inactivity.
If your are going to reopen it let us know, we will reassign your name

**Comment by rao123dk** [comment]
> We have few contributors taken up the page and we have not received the PR. I am listing download those pages, if those contributors are working on it please ping here or we will have to make those pages available for other contributors
> 
> Basics > Thinking in React (@apoorvtomar2222 )
> API Reference > React (@manavshrivastavagit )
> API Reference > DOM Elements (@rao123dk )
> Installation > Add React to a Website (@realdeepak )
> Hooks > Hooks at a Glance (@gautam-pahuja )
> Warnings > dont-call-proptypes (@hozefaj )
> FAQ > APIs and AJAX (@helios1101 )
> FAQ > Styling and CSS (@PriyankaCodes99 )

I am working on that.

**Comment by arshadkazmi42** [comment]
@rao123dk Thank you for confirmation
@gautam-pahuja Thank you for re-confirmation, I think, by mistake I took your name from the list 😄

**Comment by PriyankaCodes99** [comment]
I will work on my page.

**Comment by AkshayRaut1321** [comment]
I want to translate API reference : React.Component

**Comment by arshadkazmi42** [comment]
@AkshayRaut1321 Go ahead, I will assign your name to the page

**Comment by tibabit** [comment]
I am taking up **Introducing Hooks**

**Comment by AlokTakshak** [comment]
> @arshadkazmi42 Assign me Hooks at Glance and using State hooks

Can I take it now?

**Comment by arshadkazmi42** [comment]
@AlokTakshak You have a open PR. Do you want to start working on a new page? I can assign it to you if you can manage

**Comment by AlokTakshak** [comment]
> @AlokTakshak You have a open PR. Do you want to start working on a new page? I can assign it to you if you can manage
> 
>     * Edit
>       Just checked, Hooks at Glance is already taken up by another contributor.

Than using State hooks

**Comment by arshadkazmi42** [comment]
> > @AlokTakshak You have a open PR. Do you want to start working on a new page? I can assign it to you if you can manage
> > ```
> > * Edit
> >   Just checked, Hooks at Glance is already taken up by another contributor.
> > ```
> 
> Than using State hooks

Done, assigned

**Comment by arshadkazmi42** [comment]
> We have few contributors taken up the page and we have not received the PR. I am listing download those pages, if those contributors are working on it please ping here or we will have to make those pages available for other contributors
> 
> Basics > Thinking in React (@apoorvtomar2222 )
> API Reference > React (@manavshrivastavagit )
> API Reference > DOM Elements (@rao123dk ) -> `Working on it`
> Installation > Add React to a Website (@realdeepak )
> Hooks > Hooks at a Glance (@gautam-pahuja ) -> `Working on it`
> Warnings > dont-call-proptypes (@hozefaj )
> FAQ > APIs and AJAX (@helios1101 )
> FAQ > Styling and CSS (@PriyankaCodes99 ) -> `Working on it`

Hey Guys,
Please ping here if you are working on the pages. We will make the pages available for other contributors from whom ever we are not getting confirmation.

**Comment by arshadkazmi42** [comment]
> > We have few contributors taken up the page and we have not received the PR. I am listing download those pages, if those contributors are working on it please ping here or we will have to make those pages available for other contributors
> > Basics > Thinking in React (@apoorvtomar2222 )
> > API Reference > React (@manavshrivastavagit )
> > API Reference > DOM Elements (@rao123dk ) -> `Working on it`
> > Installation > Add React to a Website (@realdeepak )
> > Hooks > Hooks at a Glance (@gautam-pahuja ) -> `Working on it`
> > Warnings > dont-call-proptypes (@hozefaj )
> > FAQ > APIs and AJAX (@helios1101 )
> > FAQ > Styling and CSS (@PriyankaCodes99 ) -> `Working on it`
> 
> Hey Guys,
> Please ping here if you are working on the pages. We will make the pages available for other contributors from whom ever we are not getting confirmation.

Unassign the name of contributors from page who have not replied and does not have an open PR due to inactivity.
Please ping back here, if anyone is still interested in working on the assigned page

**Comment by AkshayRaut1321** [comment]
I am still active on React Components page. I just couldn't get some time last week. I hope to finish it by next weekend.

**Comment by arshadkazmi42** [comment]
@AkshayRaut1321 The message was for the list mentioned in quote. As its been long and we haven't received any PR. You have just taken up the page, take your time. Just keep us updated here if you are stuck, need help, or anything else.
So we will know that you are still working on it.
Thanks

**Comment by AkshayRaut1321** [comment]
@arshadkazmi42 thank you

**Comment by gautam-pahuja** [comment]
> > > We have few contributors taken up the page and we have not received the PR. I am listing download those pages, if those contributors are working on it please ping here or we will have to make those pages available for other contributors
> > > Basics > Thinking in React (@apoorvtomar2222 )
> > > API Reference > React (@manavshrivastavagit )
> > > API Reference > DOM Elements (@rao123dk ) -> `Working on it`
> > > Installation > Add React to a Website (@realdeepak )
> > > Hooks > Hooks at a Glance (@gautam-pahuja ) -> `Working on it`
> > > Warnings > dont-call-proptypes (@hozefaj )
> > > FAQ > APIs and AJAX (@helios1101 ) -> `Working on it`
> > > FAQ > Styling and CSS (@PriyankaCodes99 ) -> `Working on it`
> > 
> > 
> > Hey Guys,
> > Please ping here if you are working on the pages. We will make the pages available for other contributors from whom ever we are not getting confirmation.
> 
> Unassign the name of contributors from page who have not replied and do not have an open PR due to inactivity.
> Please ping back here, if anyone is still interested in working on the assigned page
> **Unassigned due to inactivity**

I am almost done, will open the PR soon.

**Comment by arshadkazmi42** [comment]
Sure, thanks for the update. You have already notified us, I have kept your name assigned. Thanks again

**Comment by apoorvtomar2222** [comment]
@arshadkazmi42 
Currently I am unable to work on Basics > Thinking in React. Please Unassign.

**Comment by arshadkazmi42** [comment]
> @arshadkazmi42
> Currently I am unable to work on Basics > Thinking in React. Please Unassign.

Thank you for informing @apoorvtomar2222

**Comment by chandrakantnaik789** [comment]
I wants to work in Rules of Hooks page, is there anyone who is working on that page.

**Comment by arshadkazmi42** [comment]
@chandrakantnaik789 go ahead. Assigning your name to the page

**Comment by kumari1224** [comment]
I want to work in `React Without ES6` page, please assign it to me if no one is working on it.

**Comment by arshadkazmi42** [comment]
@kumari1224 Go ahead. Assigned your name to the page

**Comment by kumari1224** [comment]
@arshadkazmi42 Thanks

**Comment by gautam-pahuja** [comment]
@arshadkazmi42 @saranshkataria 
I am finished with the translation of [(Hooks at a glance)](https://github.com/reactjs/hi.reactjs.org/pull/58)

**Comment by mayank-gangwal** [comment]
@saranshkataria @arshadkazmi42, I may take up Basics ->Thinking in React. Let me know the details - timeline and guidelines?

**Comment by SambhavSacheti** [comment]
@arshadkazmi42 Can you please review my changes to #30

**Comment by arshadkazmi42** [comment]
> @saranshkataria @arshadkazmi42, I may take up Basics ->Thinking in React. Let me know the details - timeline and guidelines?

@mayank-gangwal cool go ahead.
You can refere #2  for glossary and #23  about our review process

**Comment by arshadkazmi42** [comment]
> @arshadkazmi42 @saranshkataria
> I am finished with the translation of [(Hooks at a glance)](https://github.com/reactjs/hi.reactjs.org/pull/58)

cool. go ahead. I will assign your name to the page

**Comment by aashirwadgupta** [comment]
I wish to make changes to Forms #22 
Kindly let me know if it is still available to grab

**Comment by saranshkataria** [comment]
@aashirwadgupta you can take the work done in #22 and carry it forward, create a new PR. Assigning it to you.

**Comment by aashirwadgupta** [comment]
> @aashirwadgupta you can take the work done in #22 and carry it forward, create a new PR. Assigning it to you.

Thanks

**Comment by shob0** [comment]
I wish to translate Introducing Hooks.

**Comment by arshadkazmi42** [comment]
@shob0 Cool. Go ahead.

**Comment by aashirwadgupta** [comment]
Hi Team, 
I would like to take "**React**" translation under "**API reference**". 
Kindly let me know if I can go ahead with it.

**Comment by arshadkazmi42** [comment]
> Hi Team,
> I would like to take "**React**" translation under "**API reference**".
> Kindly let me know if I can go ahead with it.

@aashirwadgupta you have a PR in review. Lets get that closed before you start working on a new page

**Comment by ps0305** [comment]
@arshadkazmi42 & team,
i would like to rake **SyntheticEvent** for translation.
Please assign me the one.

**Comment by arshadkazmi42** [comment]
@ps0305 Cool Go ahead. I will assign you the page

**Comment by AkshayRaut1321** [comment]
Hi @arshadkazmi42  Sorry for late update. I got very busy past few months and the work was about 40% complete for the React Components page and now I finally decided to continue my translation work so I cloned my forked repo on my machine and it's running into issues. It was working earlier. If anyone can help with it then good otherwise you can unassign it maybe someone else might not face these issues on their machine.

**Comment by arshadkazmi42** [comment]
> Hi @arshadkazmi42 You can ignore my previous comment. I had issues with running it on my machine. But now it's working on gitpod.io so I am continuing my work there. Thanks

Cool. Let me know if you  still need some help with this.
For fork branch, I will suggest first sync it with hi.reactjs.org master master branch before starting to working on this.
There is a cli tool I have created to do this, you can give it a try if you want 
https://github.com/gh-conf/upfork

**Comment by anoopsingh1996** [comment]
Hi, @arshadkazmi42  and team,
I would like to take " Using the Effect Hook" translation under the "Hooks" page.
Kindly let me know if I can go ahead with it.

**Comment by arshadkazmi42** [comment]
> Using the Effect Hook

@anoopsingh1996 Go ahead. I will assign your name to the page

**Comment by visshaljagtap** [comment]
Hi @arshadkazmi42, I would like to translate "Building Your Own Hooks" page.
Please assign me the same.

**Comment by arshadkazmi42** [comment]
> Hi @arshadkazmi42, I would like to translate "Building Your Own Hooks" page.
> Please assign me the same.

Cool. I have assinged the page to you. Go ahead and send a PR

**Comment by mayankverr** [comment]
I'd like to take the Portals page under advanced guides

**Comment by arshadkazmi42** [comment]
@mayankverr Go ahead

**Comment by mayankverr** [comment]
@arshadkazmi42 just created a PR

**Comment by AkshayRaut1321** [comment]
Hi @arshadkazmi42  I am unable to spend time on translation so I have some partial work done. About 30% of the page is translated.
If you prefer I can make a pull request with whatever I have, or I can delete the forked repo.
Please suggest.

**Comment by arshadkazmi42** [comment]
@AkshayRaut1321 I think you can send a PR. We will see if someone else could take it up

**Comment by ps0305** [comment]
@arshadkazmi42: i can take `Hooks FAQ`, can you please, assign me one?

**Comment by arshadkazmi42** [comment]
> @arshadkazmi42: i can take `Hooks FAQ`, can you please, assign me one?

Page assigned

**Comment by bhatvikrant** [comment]
Hello dear maintainers @saranshkataria @arshadkazmi42, thanks for doing such a great job!   

I will be translating *Virtual DOM and Internals*. Please assign this to me 😀

**Comment by saranshkataria** [comment]
@bhatvikrant thank you. assigned 😄

**Comment by nnhhiilliisstt** [comment]
@arshadkazmi42 i would like to translate Passing Functions to Components.Please assign me on this one.

**Comment by bhatvikrant** [comment]
Hi, dear admins @saranshkataria, @arshadkazmi42 can you please assign me the translation of `Component State` under FAQ. 

Thanks😄.

**Comment by saranshkataria** [comment]
@bhatvikrant @nnhhiilliisstt done 👍

**Comment by Hrshk15** [comment]
How can I translate any page ?

**Comment by Hrshk15** [comment]
I would like to contribute to the react translation in HINDI LANGUAGE , what's the procedure to begin with it?

**Comment by bhatvikrant** [comment]
> I would like to contribute to the react translation in HINDI LANGUAGE , what's the procedure to begin with it?

HI there @Hrshk15 , You can check [this](https://github.com/reactjs/hi.reactjs.org/issues/23) issue for more information.

PS. I am not one of the official maintainers. Just trying to help.

**Comment by Hrshk15** [comment]
May I translate "Introducing JSX" .

**Comment by arshadkazmi42** [comment]
> May I translate "Introducing JSX" .

@Hrshk15 That is already done at #12 
You can take a page which is not assigned to anyone

**Comment by Hrshk15** [comment]
@arshadkazmi42  Can I translate the react section under API REFERENCE.

**Comment by arshadkazmi42** [comment]
@Hrshk15 Assigned API Reference > React page to you.

This is the page you need to translate
https://github.com/reactjs/hi.reactjs.org/blob/master/content/docs/reference-react.md

Translate the complete file in Hindi. Refer #2 for common translations.
Do not translate code blocks.
Once complete file is translated, send a PR for review.
There will be two phases of review. Once your PR gets approved from both the reviewers, your change will be merged.

For creating branch locally follow this
https://github.com/reactjs/hi.reactjs.org#create-a-branch

**Comment by Hrshk15** [comment]
I have tried  running it locally, following the steps mentioned.
But  it gives this error-

ERROR #85923 GRAPHQL

There was an error in your GraphQL query:

Cannot query field "allExampleCode" on type "Query".

Kindly suggest me what should I do.

**Comment by arshadkazmi42** [comment]
Follow the below steps

1. Clone repository
2. Run `npm install` in the root directory hi.reactjs.org
3. Run `npm start`

if you are seeing some error after this, share screenshot and error logs.

**Comment by neeraj-here** [comment]
Hi @arshadkazmi42 
I am interested in translating the **ReactDOMServer** under **API Reference**. Can I do that? If yes, please guide me with the procedure.

**Comment by saranshkataria** [comment]
@neeraj-here  assigned 👍

**Comment by syshcndr** [comment]
Hey!
I am interested in translating the JSX In Depth under Advanced Guides.

**Comment by neeraj-here** [comment]
> @neeraj-here assigned 👍

After

> Follow the below steps
> 
> 1. Clone repository
> 2. Run `npm install` in the root directory hi.reactjs.org
> 3. Run `npm start`
> 
> if you are seeing some error after this, share screenshot and error logs.
@arshadkazmi42  @saranshkataria 
I faced the same problem as by @Hrshk15 . Tried the above mentioned way but still getting a error. Here are the logs:


**HP@LAPTOP-BNE4L5IB MINGW64 ~
$ cd e:
HP@LAPTOP-BNE4L5IB MINGW64 /e
$ dir
!qhlogs.doc         CodeBlocks    GIMP\ 2                  My\ photos        System\ Volume\ Information
$RECYCLE.BIN        coursera-sub  Interactivity\ with\ JS  New\ folder       website
bootstrap           DS            It's\ me.rtf             New\ folder\ (2)  Whatsapp\ docs
cfrbackup-SSNIWRZC  DS.c          movies                   reactjs.org

HP@LAPTOP-BNE4L5IB MINGW64 /e
$ cd reactjs.org
HP@LAPTOP-BNE4L5IB MINGW64 /e/reactjs.org
$ cd hi.reactjs.org
HP@LAPTOP-BNE4L5IB MINGW64 /e/reactjs.org/hi.reactjs.org (master)
$ npm start

> react-website@0.0.1 start E:\reactjs.org\hi.reactjs.org
> yarn dev

yarn run v1.22.5
$ gatsby develop -H 0.0.0.0
E:\reactjs.org\hi.reactjs.org\node_modules\gatsby-recipes\node_modules\graphql\jsutils\instanceOf.js:28
      throw new Error("Cannot use ".concat(className, " \"").concat(value, "\" from another module or realm.\n\nEnsure that there is only one instance of \"graphql\" in the node_modules\ndirectory. If different versions of \"graphql\" are the dependencies of other\nrelied on modules, use \"resolutions\" to ensure only one version is installed.\n\nhttps://yarnpkg.com/en/docs/selective-version-resolutions\n\nDuplicate \"graphql\" modules cannot be used at the same time since different\nversions may have different capabilities and behavior. The data from one\nversion used in the function from another could produce confusing and\nspurious results."));
      ^

Error: Cannot use GraphQLInputObjectType "FileInput" from another module or realm.

Ensure that there is only one instance of "graphql" in the node_modules
directory. If different versions of "graphql" are the dependencies of other
relied on modules, use "resolutions" to ensure only one version is installed.

https://yarnpkg.com/en/docs/selective-version-resolutions

Duplicate "graphql" modules cannot be used at the same time since different
versions may have different capabilities and behavior. The data from one
version used in the function from another could produce confusing and
spurious results.
    at instanceOf (E:\reactjs.org\hi.reactjs.org\node_modules\gatsby-recipes\node_modules\graphql\jsutils\instanceOf.js:28:13)
    at isInputObjectType (E:\reactjs.org\hi.reactjs.org\node_modules\gatsby-recipes\node_modules\graphql\type\definition.js:168:34)
    at typeMapReducer (E:\reactjs.org\hi.reactjs.org\node_modules\gatsby-recipes\node_modules\graphql\type\schema.js:290:41)
    at Array.reduce (<anonymous>)
    at typeMapReducer (E:\reactjs.org\hi.reactjs.org\node_modules\gatsby-recipes\node_modules\graphql\type\schema.js:285:34)
    at Array.reduce (<anonymous>)
    at new GraphQLSchema (E:\reactjs.org\hi.reactjs.org\node_modules\gatsby-recipes\node_modules\graphql\type\schema.js:145:28)
    at Object.<anonymous> (E:\reactjs.org\hi.reactjs.org\node_modules\gatsby-recipes\dist\graphql-server\server.js:223:16)
    at Module._compile (internal/modules/cjs/loader.js:1133:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1153:10)

success open and validate gatsby-configs - 0.188s
success load plugins - 4.133s
success onPreInit - 0.190s
success initialize cache - 0.045s
success copy gatsby files - 0.234s
warn gatsby-plugin-feed was initialized in gatsby-config.js without a title in a feed.
This means that the plugin will use the default feed title, which may not match your use case.
This behavior will be removed in the next major release of gatsby-plugin-feed.
For more info, check out: https://gatsby.dev/adding-rss-feed
success onPreBootstrap - 0.049s
success createSchemaCustomization - 0.407s
success Checking for changed pages - 0.006s
success source and transform nodes - 8.971s

 ERROR

Missing onError handler for invocation 'building-schema', error was 'TypeError: (0 , _graphql.isScalarType) is not a function'. Stacktrace was 'TypeError: (0 , _graphql.isScalarType) is not a function
    at TypeMapper.get (E:\reactjs.org\hi.reactjs.org\node_modules\graphql-compose\lib\TypeMapper.js:88:76)
    at TypeMapper.typeFromAST (E:\reactjs.org\hi.reactjs.org\node_modules\graphql-compose\lib\TypeMapper.js:608:17)
    at TypeMapper.typeFromAST (E:\reactjs.org\hi.reactjs.org\node_modules\graphql-compose\lib\TypeMapper.js:603:24)
    at TypeMapper.getWrapped (E:\reactjs.org\hi.reactjs.org\node_modules\graphql-compose\lib\TypeMapper.js:105:17)
    at TypeMapper.convertOutputFieldConfig (E:\reactjs.org\hi.reactjs.org\node_modules\graphql-compose\lib\TypeMapper.js:272:114)
    at resolveOutputConfigAsThunk (E:\reactjs.org\hi.reactjs.org\node_modules\graphql-compose\lib\utils\configAsThunk.js:19:41)
    at ObjectTypeComposer.getFieldConfig (E:\reactjs.org\hi.reactjs.org\node_modules\graphql-compose\lib\ObjectTypeComposer.js:300:58)
    at E:\reactjs.org\hi.reactjs.org\node_modules\graphql-compose\lib\utils\toInputObjectType.js:44:19
    at Array.forEach (<anonymous>)
    at toInputObjectType (E:\reactjs.org\hi.reactjs.org\node_modules\graphql-compose\lib\utils\toInputObjectType.js:38:14)
    at ObjectTypeComposer.getInputTypeComposer (E:\reactjs.org\hi.reactjs.org\node_modules\graphql-compose\lib\ObjectTypeComposer.js:600:84)
    at E:\reactjs.org\hi.reactjs.org\node_modules\gatsby\src\schema\types\node-interface.js:22:8
    at SchemaComposer.getOrCreateOTC (E:\reactjs.org\hi.reactjs.org\node_modules\graphql-compose\lib\SchemaComposer.js:382:54)
    at getOrCreateNodeInterface (E:\reactjs.org\hi.reactjs.org\node_modules\gatsby\src\schema\types\node-interface.js:10:37)
    at getNodeInterface (E:\reactjs.org\hi.reactjs.org\node_modules\gatsby\src\schema\types\node-interface.js:81:3)
    at createSchemaComposer (E:\reactjs.org\hi.reactjs.org\node_modules\gatsby\src\schema\schema-composer.js:15:3)
    at build (E:\reactjs.org\hi.reactjs.org\node_modules\gatsby\src\schema\index.js:104:26)
    at buildSchema (E:\reactjs.org\hi.reactjs.org\node_modules\gatsby\src\services\build-schema.ts:19:3)'

⠴ building schema**

It seems to be indefinite as nothing happens after this.
Please guide me the solution or any alternative.

**Comment by bhatvikrant** [comment]
@neeraj-here  do you have access to GitHub codespaces beta?

If not then check [this](https://dev.to/vikrantbhat/open-any-public-github-repo-in-codesandbox-in-3-seconds-2il2) blog, and open the project in code sandbox instead. Maybe this might help.

**Comment by tanishqchamola** [comment]
@saranshkataria @arshadkazmi42 
Hi, can I help with translating Test Renderer page?

**Comment by ShivamPorwal02** [comment]
Hi, Can I take **dont-call-proptypes** page for translation ?

**Comment by arshadkazmi42** [comment]
> > @neeraj-here assigned 👍
> 
> After
> 
> > Follow the below steps
> > 
> > 1. Clone repository
> > 2. Run `npm install` in the root directory hi.reactjs.org
> > 3. Run `npm start`
> > 
> > if you are seeing some error after this, share screenshot and error logs.
> > @arshadkazmi42  @saranshkataria
> > I faced the same problem as by @Hrshk15 . Tried the above mentioned way but still getting a error. Here are the logs:
> 
> **HP@LAPTOP-BNE4L5IB MINGW64 ~
> $ cd e:
> HP@LAPTOP-BNE4L5IB MINGW64 /e
> $ dir
> !qhlogs.doc CodeBlocks GIMP\ 2 My\ photos System\ Volume\ Information
> $RECYCLE.BIN coursera-sub Interactivity\ with\ JS New\ folder website
> bootstrap DS It's\ me.rtf New\ folder\ (2) Whatsapp\ docs
> cfrbackup-SSNIWRZC DS.c movies reactjs.org
> 
> HP@LAPTOP-BNE4L5IB MINGW64 /e
> $ cd reactjs.org
> HP@LAPTOP-BNE4L5IB MINGW64 /e/reactjs.org
> $ cd hi.reactjs.org
> HP@LAPTOP-BNE4L5IB MINGW64 /e/reactjs.org/hi.reactjs.org (master)
> $ npm start
> 
> > react-website@0.0.1 start E:\reactjs.org\hi.reactjs.org
> > yarn dev
> 
> yarn run v1.22.5
> $ gatsby develop -H 0.0.0.0
> E:\reactjs.org\hi.reactjs.org\node_modules\gatsby-recipes\node_modules\graphql\jsutils\instanceOf.js:28
> throw new Error("Cannot use ".concat(className, " "").concat(value, "" from another module or realm.\n\nEnsure that there is only one instance of "graphql" in the node_modules\ndirectory. If different versions of "graphql" are the dependencies of other\nrelied on modules, use "resolutions" to ensure only one version is installed.\n\nhttps://yarnpkg.com/en/docs/selective-version-resolutions\n\nDuplicate "graphql" modules cannot be used at the same time since different\nversions may have different capabilities and behavior. The data from one\nversion used in the function from another could produce confusing and\nspurious results."));
> ^
> 
> Error: Cannot use GraphQLInputObjectType "FileInput" from another module or realm.
> 
> Ensure that there is only one instance of "graphql" in the node_modules
> directory. If different versions of "graphql" are the dependencies of other
> relied on modules, use "resolutions" to ensure only one version is installed.
> 
> https://yarnpkg.com/en/docs/selective-version-resolutions
> 
> Duplicate "graphql" modules cannot be used at the same time since different
> versions may have different capabilities and behavior. The data from one
> version used in the function from another could produce confusing and
> spurious results.
> at instanceOf (E:\reactjs.org\hi.reactjs.org\node_modules\gatsby-recipes\node_modules\graphql\jsutils\instanceOf.js:28:13)
> at isInputObjectType (E:\reactjs.org\hi.reactjs.org\node_modules\gatsby-recipes\node_modules\graphql\type\definition.js:168:34)
> at typeMapReducer (E:\reactjs.org\hi.reactjs.org\node_modules\gatsby-recipes\node_modules\graphql\type\schema.js:290:41)
> at Array.reduce ()
> at typeMapReducer (E:\reactjs.org\hi.reactjs.org\node_modules\gatsby-recipes\node_modules\graphql\type\schema.js:285:34)
> at Array.reduce ()
> at new GraphQLSchema (E:\reactjs.org\hi.reactjs.org\node_modules\gatsby-recipes\node_modules\graphql\type\schema.js:145:28)
> at Object. (E:\reactjs.org\hi.reactjs.org\node_modules\gatsby-recipes\dist\graphql-server\server.js:223:16)
> at Module._compile (internal/modules/cjs/loader.js:1133:30)
> at Object.Module._extensions..js (internal/modules/cjs/loader.js:1153:10)
> 
> success open and validate gatsby-configs - 0.188s
> success load plugins - 4.133s
> success onPreInit - 0.190s
> success initialize cache - 0.045s
> success copy gatsby files - 0.234s
> warn gatsby-plugin-feed was initialized in gatsby-config.js without a title in a feed.
> This means that the plugin will use the default feed title, which may not match your use case.
> This behavior will be removed in the next major release of gatsby-plugin-feed.
> For more info, check out: https://gatsby.dev/adding-rss-feed
> success onPreBootstrap - 0.049s
> success createSchemaCustomization - 0.407s
> success Checking for changed pages - 0.006s
> success source and transform nodes - 8.971s
> 
> ERROR
> 
> Missing onError handler for invocation 'building-schema', error was 'TypeError: (0 , _graphql.isScalarType) is not a function'. Stacktrace was 'TypeError: (0 , _graphql.isScalarType) is not a function
> at TypeMapper.get (E:\reactjs.org\hi.reactjs.org\node_modules\graphql-compose\lib\TypeMapper.js:88:76)
> at TypeMapper.typeFromAST (E:\reactjs.org\hi.reactjs.org\node_modules\graphql-compose\lib\TypeMapper.js:608:17)
> at TypeMapper.typeFromAST (E:\reactjs.org\hi.reactjs.org\node_modules\graphql-compose\lib\TypeMapper.js:603:24)
> at TypeMapper.getWrapped (E:\reactjs.org\hi.reactjs.org\node_modules\graphql-compose\lib\TypeMapper.js:105:17)
> at TypeMapper.convertOutputFieldConfig (E:\reactjs.org\hi.reactjs.org\node_modules\graphql-compose\lib\TypeMapper.js:272:114)
> at resolveOutputConfigAsThunk (E:\reactjs.org\hi.reactjs.org\node_modules\graphql-compose\lib\utils\configAsThunk.js:19:41)
> at ObjectTypeComposer.getFieldConfig (E:\reactjs.org\hi.reactjs.org\node_modules\graphql-compose\lib\ObjectTypeComposer.js:300:58)
> at E:\reactjs.org\hi.reactjs.org\node_modules\graphql-compose\lib\utils\toInputObjectType.js:44:19
> at Array.forEach ()
> at toInputObjectType (E:\reactjs.org\hi.reactjs.org\node_modules\graphql-compose\lib\utils\toInputObjectType.js:38:14)
> at ObjectTypeComposer.getInputTypeComposer (E:\reactjs.org\hi.reactjs.org\node_modules\graphql-compose\lib\ObjectTypeComposer.js:600:84)
> at E:\reactjs.org\hi.reactjs.org\node_modules\gatsby\src\schema\types\node-interface.js:22:8
> at SchemaComposer.getOrCreateOTC (E:\reactjs.org\hi.reactjs.org\node_modules\graphql-compose\lib\SchemaComposer.js:382:54)
> at getOrCreateNodeInterface (E:\reactjs.org\hi.reactjs.org\node_modules\gatsby\src\schema\types\node-interface.js:10:37)
> at getNodeInterface (E:\reactjs.org\hi.reactjs.org\node_modules\gatsby\src\schema\types\node-interface.js:81:3)
> at createSchemaComposer (E:\reactjs.org\hi.reactjs.org\node_modules\gatsby\src\schema\schema-composer.js:15:3)
> at build (E:\reactjs.org\hi.reactjs.org\node_modules\gatsby\src\schema\index.js:104:26)
> at buildSchema (E:\reactjs.org\hi.reactjs.org\node_modules\gatsby\src\services\build-schema.ts:19:3)'
> 
> ⠴ building schema**
> 
> It seems to be indefinite as nothing happens after this.
> Please guide me the solution or any alternative.

try delete node-modules folder and package-lock.json or yarn.lock then try running `yarn` and then `yarn dev`

**Comment by arshadkazmi42** [comment]
> @saranshkataria @arshadkazmi42
> Hi, can I help with translating Test Renderer page?

sure @tanishqchamola . page assigned

**Comment by arshadkazmi42** [comment]
> Hi, Can I take **dont-call-proptypes** page for translation ?

@ShivamPorwal02 page assigned

**Comment by vivek80801** [comment]
How can I contribute in translation of reactjs in hindi.?

**Comment by vivek80801** [comment]
Can I also help in translation of reactjs?

**Comment by vieee** [comment]
Hi, can I help with translating the `Hooks API reference` page within the Hooks section..?

**Comment by piyush9311** [comment]
I would like to work upon the issue #26 and translate "State and Lifecycle".

**Comment by Himnashu** [comment]
Hi @arshadkazmi42, can I help with translating the Reconciliation page?

**Comment by vishalnagda1** [comment]
@tesseralis I would like to translate **React Without JSX** page in **Advanced Guides**.
Please assign it to me.

**Comment by arshadkazmi42** [comment]
> @tesseralis I would like to translate **React Without JSX** page in **Advanced Guides**.
> 
> Please assign it to me.

@vishalnagda1 Go ahead

**Comment by vishalnagda1** [comment]
@arshadkazmi42 Completed the last task. Please review the PR #163
Next I would like to take **JSX In Depth** page in **Advanced Guides** for translation.

**Comment by vishalnagda1** [comment]
@arshadkazmi42 Should I proceed with *JSX In Depth* page in *Advanced Guides* for translation, or should I wait for previous PR to get merge before starting with new one?

**Comment by arshadkazmi42** [comment]
@vishalnagda1 sure go ahead

**Comment by gaearon** [comment]
Thanks to everyone who contributed to this iteration!
Here is the issue for the new website translation: https://github.com/reactjs/hi.react.dev/issues/168.
